modules/distribution/resources/api_templates/endpoint_template.xml

```diff
@@ -128,17 +131,31 @@ optimize="$advance_ep.get("optimize")"
         #macro( draw_endpoint $type $endpoint_config )
 
         #set( $endpointClass = $endpoint_config.get("endpoint_type") )
-        #set( $endpoints = $endpoint_config.get("${type}_endpoints"))
-        #set( $ep_key = "${endpointKey}_API${type}Endpoint")
-
-        #if ( $endpointClass == "http" )
-        #http_endpoint( $ep_key $endpoints )
-        #elseif( $endpointClass == "ws")
-        #websocket_endpoint( $ep_key )
-        #elseif ( $endpointClass == "address" )
-        #address_endpoint($ep_key $endpoints)
-        #elseif ( $endpointClass == "failover" )
-        #set( $failover_endpoints = $endpoint_config.get("${type}_failovers"))
+
+        ##If graphql api
+        #if( $endpointClass == "graphql" )
```

Comment:
Code modification related to GQL Subscriptions
`
        ##If graphql api
        #if( $endpointClass == "graphql" )
            #if( $websocketResourceKey )
                ##If websocketResourceKey present in context then this is the graphQL subscription endpoint
                #set( $endpoint_config = $endpoint_config.get("ws") )
                #set( $isGQLSubscription = "true" )
            #else
                ##Else GraphQL query mutation http endpoint
                #set( $endpoint_config = $endpoint_config.get("http") )
            #end
            #set( $endpointClass = $endpoint_config.get("endpoint_type") )
        #end`